cmd/neofs-node/config.go

```diff
@@ -743,6 +743,11 @@ func initBasics(c *cfg, key *keys.PrivateKey, stateStorage *state.PersistentStor
 
 	eDuration, err := nmWrap.EpochDuration()
 	fatalOnErr(err)
+	if eDuration == 0 {
```

Comment:
It's all in `neofs-node`, likely you can have some `nState.UpdateEpochDuration()` that will handle this.

Reply:
ok, degraded logs a little then